docs/examples/embeddings/Anyscale.ipynb

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
<p>could you show case for a simple embedding for a query</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9470/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>